website/docs/components/table/partials/guidelines/guidelines.md

```diff
@@ -38,16 +58,55 @@ Use consistent alignment throughout the Table. We recommend using end-alignment
 ![Table column placement example](/assets/components/table/table-alignment-dont.png =500x*)
 !!!
 
-### Placement
+### Start alignment
 
-!!! Info
+Align content to the start of the cell by default. This ensures readability across different content types, consistency in content of varying lengths, and alignment between the column header label and the content within the cell.
+
+Use start alignment for:
+
+- String and text-based content (unique identifiers or IDs, names and naming conventions, etc).
+- Numerical values that do not contain decimals or floating point numbers.
+- Numerical values that contain periods or other delimeter characters (IP addresses).
+- Nested components that display a string or text value, e.g., a [Badge](/components/badge).
+
+![Start alignment of content within a table](/assets/components/table/start-alignment-example.png)
+
+### End alignment
+
+End alignment can be used when expressing numerical values with decimals as this aligns the decimal places vertically.
+
+Common examples of end alignment include:
+
+- Financial information and currency amounts.
+- Fractional and floating point values represented with decimals.
+
+![End alignment of content within a table](/assets/components/table/end-alignment-example.png)
+
+End alignment can also be used in the last column of a table to:
+
+- Highlight a "more options" function pertaining to the content within a row.
+- As a means to visually "bookend" the row with content that is of a similar length, e.g., timestamps, TTL (time-to-live) values, dates.
+
+![End alignment example within a table with a date and more options](/assets/components/table/end-alignment-example-02.png)
+
+!!! Dont
+
+Don't end align content that is variable in length. This can make the content more difficult to read by forcing an unnatural [reading pattern](patterns/button-organization?tab=research#layout-and-reading-patterns).
```

Comment:
```suggestion
Don’t end align content that is variable in length. This can make the content more difficult to read by forcing an unnatural [reading pattern](patterns/button-organization?tab=research#layout-and-reading-patterns).
```